app/views/complaints/_complaints_table.html.erb

```diff
@@ -1,14 +1,19 @@
-<table class="usa-table" aria-describedby="#caption" >
+<table class="usa-table">
+  <caption>
+    Complaints associated with this user. Data for illustration purposes only.
+  </caption>
   <thead>
     <tr>
       <th scope="col" role="columnheader">HSES Issue #</th>
       <th
         data-sortable
         scope="col"
         role="columnheader"
+        <%= sanitize aria_sort("creationDate") %>
+        data-next-sort="<%= complaints_path(page: 1, sort: (current_sort_direction("creationDate") == "descending" ? "creationDate" : "-creationDate")) %>"
```

Comment:
[suggestion, nonblocking] If you feel like you have the will and time to work on this, I'd prefer that this sort of logic was happening in a helper rather than directly in the view.